docs/business-apps/power-automate/get-started/sharepoint-actions-triggers.md

```diff
@@ -129,47 +129,47 @@ Gets all lists and libraries.
 
 ### Get file content
 
-Gets file contents using the file identifier. The contents can be copied somewhere else, or be used as an attachment.
+Gets file contents using the file identifier. You can copy the contents somewhere else, or use it as an attachment.
 
 ### Get file content using path
 
-Gets file contents using the file path.
+Gets file contents, using the file path.
 
 ### Get file metadata
 
-Gets info about the file, such as size, etag, created date, and so on. Uses a file identifier to pick the file. To get to the values stored in the columns in the library, use "Get file properties" action.
+Gets info about the file, such as size, etag, created date, and so on. Uses a file identifier to select the file. To get to the values stored in the columns in the library, use the **Get file properties** action.
 
 ### Get file metadata using path
 
-Gets info about the file, such as size, etag, created date, and so on. Uses a file path to pick the file. To get to the values stored in the columns in the library, use "Get file properties" action.
+Gets info about the file, such as size, etag, created date, and so on. Uses a file path to select the file. To get to the values stored in the columns in the library, use the **Get file properties** action.
 
 ### Get file properties
 
 Gets the properties saved in the columns in the library for the item specified by the item id. To get to the contents of the file, add a "Get file content" step, and use the "File identifier" property returned by this action. When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to.
 
 ### Get files (properties only)
 
-Gets the properties saved in the columns in the library for all folders and files stored in the library. You can also filter down to the items that match a condition. Use "Apply to each" section to work with the output from this action. When using this with the On-Premises Data Gateway, you may need to be manually enter the name of the library to connect to.
+Gets the properties saved in the columns in the library for all folders and files stored in the library. You can also filter down to the items that match a condition. To work with the output from this action, use the **Apply to each** section. When using this with the On-Premises Data Gateway, you may need to be manually enter the name of the library to connect to.
 
 ### Get folder metadata
 
-Gets info about the folder. Uses a file identifier to pick the folder.
+Gets info about the folder. Uses a file identifier to select the folder.
 
 ### Get folder metadata using path
 
-Gets info about the folder. Uses a folder path to pick the folder.
+Gets info about the folder. Uses a folder path to select the folder.
 
 ### Get item
 
-Gets a single item by its id from a SharePoint list.
+Gets a single item by its ID from a SharePoint list.
 
 ### Get items
```

Comment:
Missing description. We can add the following:

_Gets items from a SharePoint list. You can specify to get items from a specific folder and/or its sub-folders. To limit items to certain items, you can also specify filter queries. To learn more, visit this detailed documentation on how to use Get items action._

@jackwi-alt  the last sentence should link to the 'Working with Get items and Get Files' document.